koku/api/report/ocp/provider_map.py

```diff
@@ -71,7 +71,6 @@ def __init__(self, provider, report_type):
                                     KeyDecimalTransform("storage", "supplementary_usage_cost"),
                                     Value(0, output_field=DecimalField()),
                                 )
-                                + Coalesce(F("supplementary_monthly_cost"), Value(0, output_field=DecimalField()))
```

Comment:
`supplementary_monthly_cost` and `infrastructure_monthly_cost` are old/unused fields that were not transferred over to the new partables.